app/controllers/users_controller.rb

```diff
@@ -42,6 +45,12 @@ def update
       end
     end
 
+    no_flash_errors = (!flash[:errors] || flash[:errors].size.zero?)
+
+    if @user.valid? && no_flash_errors && review_required
```

Comment:
Just curious why the user might _not_ be valid here?

Reply:
That's up to the user model to determine. Something left blank ? I was just preserving the original behaviour.